agenta-web/src/components/Playground/NewVariantModal.tsx

```diff
@@ -41,10 +41,10 @@ const NewVariantModal: React.FC<Props> = ({
         setIsInputValid(newVariantName.trim().length > 0 && value !== "Source Variant")
     }
 
-    const handleInputChange = (e: React.ChangeEvent<HTMLInputElement>) => {
-        const inputValue = e.target.value
-        setNewVariantName(inputValue)
-        setIsInputValid(inputValue.trim().length > 0 && variantPlaceHolder !== "Source Variant")
+    const handleVariantNameChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+        const variantName = e.target.value
+        setNewVariantName(variantName)
+        setIsInputValid(variantName.trim().length > 0) // Update the validity of the input
```

Comment:
Why did you remove `&& variantPlaceHolder !== "Source Variant")`?

That line makes sure we don't enable the button if the user did not select yet a source variant

Reply:
My apologies, I accidentally removed it. Thanks for catching that; I've fixed it now!

Reply:
Thanks @yeokyeong-yanolja